CONTRIBUTING.md

```diff
@@ -45,6 +61,7 @@ For each new detector, at least one regression tests must be present.
 - Create a test in `tests`
 - Update `ALL_TEST` in `tests/test_detectors.py`
 - Run `python ./tests/test_detectors.py --generate`. This will generate the json artifacts in `tests/expected_json`. Add the generated files to git.
+    - If updating an existing detector, either delete the old json artifacts or run `python ./tests/test_detectors.py --generate` instead.
```

Comment:
"delete the old json artifacts", this might be confusing as we don't want people to delete all the json artifacts - this would lead to a lot of unnecessary changes in the commit

Maybe `denfity the json artifacts related to your changes, and delete them` , or something like that?